### PR TITLE
Add a fixes tool for adding nodes from missing autoadds (SYN-2343)

### PR DESCRIPTION
**Pull request by vEpiphyte**
Related to #2170

### CODE REVIEW

File: synapse/tools/fixes/autoadds.py [hunk 1]
```diff
@@ -0,0 +1,190 @@
+import sys
```
Comment (invisig0th) [inline synapse/tools/fixes/autoadds.py:1]: I was kinda hoping this could be something we did in pure storm.  Anything that would prevent that?

Comment (vEpiphyte) [reply synapse/tools/fixes/autoadds.py:1]: We could possibly to that. Building the view tree so we're fixing forked views in order may be a little interesting. Would this then be a pure command then?

Comment (invisig0th) [reply synapse/tools/fixes/autoadds.py:1]: Probably a lib function ( maybe that could even check a version global and do incremental updates from one place in the future )

File: synapse/tests/test_lib_stormlib_cell.py [hunk 2]
```diff
@@ -21,6 +23,12 @@ async def test_stormlib_cell(self):
             ret = await core.callStorm('return ( $lib.cell.getHealthCheck() )')
             self.eq(ret, await core.getHealthCheck())
 
+            # New cores have stormvar set to teh current max version fix
```
Comment (jnwatson) [inline synapse/tests/test_lib_stormlib_cell.py:26]: teh

**Review by jnwatson — DISMISSED** [review]

File: synapse/lib/stormlib/cell.py [hunk 3]
```diff
@@ -28,8 +131,63 @@ def getObjLocals(self):
...
+            else:
+                await self.runt.snap.core.setStormVar(runtime_fixes_key, vers)
+                await self.runt.printf(f'Applied fix {vers}')
+            curv = vers
+
+        return curv
+
+    async def _stormFixesCheck(self):
+        if not self.runt.isAdmin():
+            mesg = '$lib.cell.getHealthCheck() requires admin privs.'
```
Comment (Cisphyx) [inline synapse/lib/stormlib/cell.py:175]: Error message needs updating

File: synapse/lib/stormlib/cell.py [hunk 4]
```diff
@@ -1,6 +1,102 @@
+import asyncio
+import logging
+
 import synapse.exc as s_exc
+import synapse.common as s_common
```
Comment (Cisphyx) [inline synapse/lib/stormlib/cell.py:5]: Doesn't look `s_common` ended up being used

**Review by Cisphyx — APPROVED** [review]

**Pull request merged**